### DISCUSSION TITLE
Date Picker but month and year only

### DISCUSSION THREAD

**Discussion by florianwachs** [microsoft/fluentui-blazor#1112] [Q&A] [answered]
I really love that i can change the calendar to only select month and year. is this behavior also possible with the date picker?

thank you very much

**Comment by dvoituron** [comment, marked as answer]
Hello. 

Not directly using the DatePicker, because you have to select a date. 

You could use the FluentCalendar, using the Month view, or two FluentSelect with months and years. 

May be in a future release, we could add a View attribute.
